Core supplies the binary; luci-app-adguardhome supplies service files and conffiles.

--- a/feeds/packages/net/adguardhome/Makefile
+++ b/feeds/packages/net/adguardhome/Makefile
@@ -46,11 +46,6 @@
 	USERID:=adguardhome=853:adguardhome=853
 endef
 
-define Package/adguardhome/conffiles
-/etc/adguardhome/adguardhome.yaml
-/etc/config/adguardhome
-endef
-
 define Package/adguardhome/description
 Free and open source, powerful network-wide ads and trackers blocking DNS server.
 endef
@@ -70,20 +65,6 @@
 
 define Package/adguardhome/install
 	$(call GoPackage/Package/Install/Bin,$(1))
-	$(INSTALL_DIR) $(1)/etc/capabilities
-	$(INSTALL_CONF) ./files/adguardhome.json $(1)/etc/capabilities/adguardhome.json
-
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/adguardhome.conf $(1)/etc/config/adguardhome
-
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/adguardhome.init $(1)/etc/init.d/adguardhome
-
-	$(INSTALL_DIR) $(1)/etc/sysctl.d
-	$(INSTALL_CONF) ./files/adguardhome.sysctl $(1)/etc/sysctl.d/50-adguardhome.conf
-
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(INSTALL_BIN) ./files/adguardhome.defaults $(1)/etc/uci-defaults/adguardhome
 endef
 
 $(eval $(call Download,adguardhome-frontend))
